### ISSUE TITLE
About third-party date and time widgets

### ISSUE THREAD

**Issue by fengzhihenxs** [surveyjs/widgets#198]
Now the officially supported third-party date widget is bootstrap datepicker, but bootstrap datepicker cannot set the time part. Is there a way to support bootstrap datetimepicker, so that a widget date, time, datetime can be supported.

**Comment by tsv2013** [comment]
Yes, you can implement a custom date time picker widget using the https://github.com/surveyjs/widgets/blob/master/src/bootstrapdatepicker.js widget code as a basis.

**Comment by fengzhihenxs** [comment]
@tsv2013 Thank you very much, I have implemented bootstrap datatimepicker.
After the implementation, there is a small problem that needs to be asked to you: When the scroll bar appears in the survey container, scroll the scroll bar of the survey container, the datatimepicker window is fixed on the current page, and does not move with the scroll bar, but when the survey container does not appear the scroll bar When scrolling the scroll bar of the survey container, the datatimepicker window moves with the scroll bar. I think this is not reasonable. Is there a solution?

This is an example：https://plnkr.co/edit/lPfGBzQpEa98Gbch?preview

**Comment by fengzhihenxs** [comment]
Just now is the bootstrap datepicker example, which is also true. The following is the case of bootstrap datatimepicker.
![image](https://user-images.githubusercontent.com/37610470/84623879-d2da1080-af12-11ea-802c-e6758c90ae0e.png)

**Comment by tsv2013** [comment]
@fengzhihenxs for some reason I think it’s caused by bootstrap... propable popup container is misplaced

**Comment by sirajansari3** [comment]
@tsv2013  Hi,
I am trying to add one of the custom widget to survey creator toolbox but getting failed...plz help.
i m referring this example https://codesandbox.io/s/crazy-rain-3wqj7?file=/src/customwidget.js

**Comment by tsv2013** [comment]
Answered here - https://surveyjs.answerdesk.io/ticket/details/t5959/unable-to-add-custom-widget-to-creator-toolbox

**Comment by Mahesh4Providence** [comment]
Hello @fengzhihenxs ,
Were you able to impliment a custom bootstrap datatime picker. (not datepicker).
If Yes, Could you share the widget.js code

**Comment by Mahesh4Providence** [comment]
Hello @fengzhihenxs ,
Could you share the widget.js code for bootstrap datatimepicker .
